uimaj-core/src/main/java/org/apache/uima/jcas/tcas/Annotation.java

```diff
@@ -244,4 +245,39 @@ public final int compareAnnotationWithId(Annotation other, LinearTypeOrder lto)
     return Integer.compare(_id,  other._id);
   }
 
+  @Override
+  public void trim(IntPredicate aIsTrimChar) {
```

Comment:
Maybe one more case to cover: getDocumentText could return null (if never set).

Reply:
That would then thrown a NullPointerException - which sounds reasonable - any better idea what it should do?

Reply:
That would be fine, or just returning could be an alternative (since there's nothing to trim).